.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+image: ubuntu
```

Comment:
Could you please move this file into your folder

Reply:
Done